pftop.c

```diff
@@ -730,6 +730,12 @@ read_states(void)
 		cache_endupdate();
 	}
 
+	while (!TAILQ_EMPTY(&ps.states)) {
```

Comment:
You probably want to just do `pfctl_free_states(pf_dev, &ps);` instead. That's both shorter and means that if libpfctl ever starts allocating anything else in struct pfctl_states it'll get cleaned up too without having to change pftop.

Reply:
@kprovost Thanks!